5월2일/1388/boj1388_java_유승훈.java

```diff
@@ -0,0 +1,65 @@
+package Ex1388;
+
+import java.util.Scanner;
+
+public class Main {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		Scanner sc = new Scanner(System.in);
+		int N = sc.nextInt();
+		int M = sc.nextInt();
+		
+		char graph[][] = new char[N][M];
+		
+		for(int i=0; i<N; i++)
+		{
+			String str = sc.next();
+			char[] arr = str.toCharArray();
```

Comment:
17~18번째 라인을 아래와 같이 바꿔보는건 어떤가요?
```suggestion
			graph[i] = sc.nextLine().toCharArray();
```
bufferedReader에도 동일합니다~